index.html

```diff
@@ -0,0 +1,578 @@
+  - [**Standard rules for extensibility**](#extensibility) of directives and their associated input types. These rules allow specs to annotate each others' metadata, effectively providing "directives on directives," which are otherwise impossible within the GraphQL schema language
+
+**Core schemas are not a new language.** All core schema documents are valid GraphQL schema documents. However, this specification introduces new requirements, so not all valid GraphQL schemas are valid core schemas.
+
+The broad intention behind core schemas is to provide a *single document* which provides all the necessary configuration of a *data core*—some program which serves the schema to GraphQL clients, primarily by following directives in order to determine how to resolve queries made against that schema.
+
+### Parts of a Core Schema
+
+When talking about a core schema, we can broadly break it into two pieces:
+- an **export schema**, which contains schema elements (objects, interfaces, enums, directives, etc.) which **should** be served to clients, and
+- **machinery** consisting of directives and associated input types (such as enums and input objects). Machinery **must not** be served to clients.
+
+## Roles
+
+```mermaid
+#::[Actors who may be interested in the core schemas]
+graph TB
+  author("👩🏽‍💻 🤖  &nbsp;Author")-->schema(["☉ Core Schema"])
+  schema-->proc1("🤖 &nbsp;Processor")
+  proc1-->output1(["☉ Core Schema[0]"])
+  output1-->proc2("🤖 &nbsp;Processor")
+  proc2-->output2(["☉ Core Schema[1]"])
+  output2-->etc("...")
+  etc-->final(["☉ Core Schema [final]"])
+  final-->core("🤖 Data Core")
+  schema-->reader("👩🏽‍💻  Reader")
+  output1-->reader
+  output2-->reader
+  final-->reader
+```
+
+- **Authors (either human or machine)** write an initial core schema as specified in this document, including versioned `@core` requests for all directives they use
+- **Machine processors** can process core schemas and output new core schemas. The versioning of directives and associated schema elements provided by the `@core` allows processors to operate on directives they understand and pass through directives they do not.
+- **Human readers** can examine the core schema at various stages of processing. At any stage, they can examine the `@core` directives and follow URLs to the specification, receiving an explanation of the requirements of the specification and what new directives, types, and other schema objects are available within the document.
+- **Data cores** can then pick up the processed core schema and provide some data-layer service with it. Typically this means serving the export schema as a GraphQL endpoint, using metadata defined by machinery to inform how it processes operations it receives. However, data cores may perform other tasks described in the core schema, such as routing to backend services, caching commonly-accessed fields and queries, and so on. The term "data core" is intended to capture this multiplicity of possible activities.
```

Comment:
FWIW I don't really find the concept of a "data core" (as opposed to just "GraphQL server") to be particularly helpful. I don't object to us keeping it in the spec, just a data point.